helm/metac/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+name: metac
+apiVersion: v1
+appVersion: "v0.1.0"
+description: A Helm chart for Metac
+version: 0.1.0
+home: https://github.com/luisdavim/metac
+keywords:
+- CRDs
+- metac
+- operator
+- controller
+- framework
+- metacontroller
+sources:
+- https://github.com/AmitKumarDas/metac
```

Comment:
Do we need to have multiple links?
e.g.
https://github.com/luisdavim/metac
& https://github.com/AmitKumarDas/metac

Reply:
Oh, that was a mistake, I'll fix it.